app/main.py

```diff
@@ -4,4 +1,39 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_weather() -> str:
+    api_key = os.getenv("API_KEY")
+    city = os.getenv("CITY")
+    if not api_key:
+        raise ValueError("API_KEY is not set")
+    if not city:
+        raise ValueError("CITY is not set")
+
+    url = "http://api.weatherapi.com/v1/current.json?"
+    response = requests.get(url + f"key={api_key}&q={city}")
+    if response.status_code == 200:
+        return (
+            f"{response.json()['location']['name']}/"
+            f"{response.json()['location']['country']} "
+            f"{response.json()['location']['localtime']} "
+            f"Weather: {response.json()['current']['temp_c']} Celsius, "
+            f"{response.json()['current']['condition']['text']}"
```

Comment:
There is a syntax error in the string formatting due to the use of double quotes inside double quotes. Consider using single quotes for the string literals or escaping the double quotes to fix this issue.